tests/NClient.Providers/NClient.Providers.Mapping.HttpResponses.Tests/HttpResponseBuilderTest.cs

```diff
@@ -272,5 +272,59 @@ public async Task Build_FailureHttpResponse_HttpResponseWithDataAndError()
             ((HttpResponseWithError<BasicEntity, string>) actualResult!).StatusCode.Should().Be(httpResponseMessage.StatusCode);
             ((HttpResponseWithError<BasicEntity, string>) actualResult!).Error.Should().Be(expectedError);
         }
+        
+        [Test]
+        public async Task Build_FailureHttpResponse_HttpResponseWithDataAndError_Should_Deconstruct()
```

Comment:
In general, I wanted simple tests where a response instance was created using a constructor, and then a deconstruct method was called :)

Reply:
This is working too. What is the problem?)

Reply:
Currently, only `HttpResponse*` is being tested, but there are also `Result` and `Response`. In addition, not all cases for `HttpResponse*` are covered. Of course, you can do the same for all classes and cases, but it's difficult, it will be easier to make simple unit tests for classes with deconstruct methods, avoiding integration tests

Reply:
Ok.